spring-cloud-starter-dataflow-server-kubernetes/pom.xml

```diff
@@ -21,5 +21,51 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-dataflow-server-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.mariadb.jdbc</groupId>
+			<artifactId>mariadb-java-client</artifactId>
+			<scope>runtime</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+			<scope>runtime</scope>
+		</dependency>
+		<!-- Fabric8 spring-cloud-kubernetes dependencies -->
+		<dependency>
```

Comment:
Actually, shouldn't the following 3 deps be in the deployer itself alone? Or are you suffering from the same problem as the CF-server wrt reactor (I doubt it)